src/main/java/core/basesyntax/Excavator.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+
+public class Excavator extends Machine {
+    public void doWork() {
+        System.out.println("Excavator has started working.");
+    }
+
+    @Override
+    public void stopWork() {
+        System.out.println("Excavator has stopped working.");
```

Comment:
The message in the `stopWork` method should be more informative and indicate the type of machine that has stopped working. For example: 'Excavator has stopped working.'